upgrade/index.md

```diff
@@ -14,6 +14,12 @@ product streams for the HPE Cray EX system can be installed or upgraded.
 Note: If problems are encountered during the upgrade, some of the topics do have their own troubleshooting
 sections, but there is also a general troubleshooting topic.
 
+## Known Issues
+
+- kdump (kernel dump) may hang and not work in 1.2 (CASMTRIAGE-3591)[https://jira-pro.its.hpecorp.net:8443/browse/CASMTRIAGE-3591]
```

Comment:
We should remove the Jira ticket number since that has no meaning to our customers.

This should read:

"kdump (kernel dump) may hang and fail on NCNs in CSM 1.2 (Raspberry)"

Reply:
Got it.